tests/itests/pom.xml

```diff
@@ -0,0 +1,229 @@
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.0.1</version>
+                <executions>
+                    <execution>
+                        <id>unpack-ddf-common</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.codice.ddf</groupId>
+                                    <artifactId>ddf-common</artifactId>
+                                    <version>${ddf.version}</version>
+                                    <type>jar</type>
+                                    <outputDirectory>${project.build.testOutputDirectory}/ddf-common</outputDirectory>
+                                    <excludes>**/META-INF/**,**/OSGI-INF/**</excludes>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.codice.ddf.admin.query</groupId>
+                                    <artifactId>admin-query-common</artifactId>
+                                    <version>${project.version}</version>
+                                    <type>jar</type>
+                                    <outputDirectory>${project.build.testOutputDirectory}</outputDirectory>
+                                    <excludes>**/META-INF/**,**/OSGI-INF/**</excludes>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.codice.ddf.admin.query</groupId>
+                                    <artifactId>admin-query-ldap</artifactId>
+                                    <version>${project.version}</version>
+                                    <type>jar</type>
+                                    <outputDirectory>${project.build.testOutputDirectory}</outputDirectory>
+                                    <excludes>**/META-INF/**,**/OSGI-INF/**</excludes>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.codice.ddf.admin.query</groupId>
+                                    <artifactId>admin-query-security-common</artifactId>
+                                    <version>${project.version}</version>
+                                    <type>jar</type>
+                                    <outputDirectory>${project.build.testOutputDirectory}</outputDirectory>
+                                    <excludes>**/META-INF/**,**/OSGI-INF/**</excludes>
+                                </artifactItem>
```

Comment:
Thanks for trying this out! I added these dependencies to the test-commons pom.xml instead.